### ISSUE TITLE
[Keyboard Navigation - Settings - Project Settings]Visual focus indicator is not visible on save and Editing settings as text buttons on project settings pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2566]
**User Experience:**
Users who depends on Keyboard will feel difficulty to access any controls if Focus indicator is not visible on that.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Settings control lying in the header section on the page and select it.
4-Navigate to the Project Settings menu option listed under the Settings menu control and select it.
5-Navigate to the various controls lying in the pane opened.
6-Verify Visual focus indicator is visible or not on ‘save and Editing settings as text’ buttons on project settings pane.

**Actual Result:**
Visual focus indicator is not visible on save and Editing settings as text buttons on project settings pane.

**Expected Result:**
Visual focus indicator should be visible on save and Editing settings as text buttons on project settings pane.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Suggested fix:**
1.Using user interface components that are highlighted by the user agent when they receive focus
2.Using CSS to change the presentation of a user interface component when it receives focus
3.Using the default focus indicator for the platform so that high visibility default focus indicators will carry over
4.Using an author-supplied, highly visible focus indicator
5.Using script to change the background color or border of the element with focus
#div a:hover, #div a:active, #div a:focus {
background-color: #DCFFFF;
color:#66;
}
To provide an outline around the element we can use the below
a { outline: none; }

**Refer attachment file for more details:** 
[MAS2.4.7_Project_Focus indicator is not visible on buttons.zip](https://github.com/Microsoft/pxt/files/1192981/MAS2.4.7_Project_Focus.indicator.is.not.visible.on.buttons.zip)

**Comment by veler** [comment]
Fixed in the following PR : [https://github.com/Microsoft/pxt/pull/2845](https://github.com/Microsoft/pxt/pull/2845)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug.
